content/administration/maintain/mailjet_api.rst

```diff
@@ -0,0 +1,213 @@
+page. Next, navigate to the :guilabel:`Senders & Domains` section and click on :guilabel:`SMTP and
+SEND API Settings`.
+
+.. image:: mailjet_api/api-settings.png
+   :align: center
+   :alt: SMTP and Send API Settings link in the Senders & Domains section of Mailjet.
+
+Then, copy the SMTP configuration settings onto a notepad. They can be found under the
+:guilabel:`Configuration (SMTP only)` section. The SMTP configuration settings include the server
+address, the security option needed (Use SSL/TLS), and the port number. The settings are needed to
+configured Mailjet in Odoo, which will be covered in the :ref:`last section
+<maintain/mailjet-api/odoo-setup>`.
+
+.. note::
+   Here is `Mailjet's SMTP Configuration documentation <https://documentation.mailjet.com/hc/en-us/
+   articles/360043229473-How-can-I-configure-my-SMTP-parameters->`_ for more information.
+
+.. note::
+   Odoo blocks `port 25` on SaaS and Odoo.sh databases.
+   :ref:`See reference here <email_servers/restriction>`.
+
+.. image:: mailjet_api/smtp-config.png
+   :align: center
+   :alt: SMTP configuration from Mailjet.
+
+Next, click on the button labeled :guilabel:`Retrieve your API credentials` to retrieve the Mailjet
+API credentials.
+
+Then, reveal the :guilabel:`API key` by clicking on the eye icon. Copy this key to a notepad, as
+this serves as the :guilabel:`Username` in the Odoo configuration. Next, generate the
+:guilabel:`Secret Key` by clicking on the :guilabel:`Generate Secret Key` button. Copy this key to
+a notepad, as this serves as the :guilabel:`Password` in the Odoo configuration.
+
+Add verified sender address(s)
+------------------------------
+
+The next step is to add a sender address or a domain to the Mailjet account settings so that the
+email address or domain is approved to send emails using Mailjet's servers. First, navigate to the
+`Mailjet Account Information <https://app.mailjet.com/account>`_ page. Next, click on
+:guilabel:`Add a Sender Domain or Address` link under the :guilabel:`Senders & Domains` section.
+
+.. image:: mailjet_api/add-domain-email.png
+   :align: center
+   :alt: Add a sender domain or address in the Mailjet interface.
+
+Determine if a sender's email address or the entire domain needs to be added to the Mailjet
+settings. By default, the email address originally set up in the Mailjet account is added as a
+trusted sender.
+
+To add another email address, click on the button labeled :guilabel:`Add a sender address`. Then,
+add the email address that is configured to send from the custom domain. It may be easier to
+configure the domain as a whole if :abbr:`DNS (Domain Name System)` access is available. Jump to
+the :ref:`Add a domain <maintain/mailjet-api/add-domain>` section for steps on adding the domain.
+
+.. note::
+   Either all email addresses of the Odoo database users who are sending emails using Mailjet's
+   servers need to be configured or the domain(s) of the users' email addresses can be configured.
```

Comment:
```suggestion
Determine if a sender's email address or the entire domain needs to be added to the Mailjet
settings. It may be easier to configure the domain as a whole if :abbr:`DNS (Domain Name System)`
access is available. Jump to the :ref:`Add a domain <maintain/mailjet-api/add-domain>` section for
steps on adding the domain. 

.. note::
   Either all email addresses of the Odoo database users who are sending emails using Mailjet's
   servers need to be configured or the domain(s) of the users' email addresses can be configured.

By default, the email address originally set up in the Mailjet account is added as a trusted sender.
To add another email address, click on the button labeled :guilabel:`Add a sender address`. Then,
add the email address that is configured to send from the custom domain.
```